File: core-java-practice/gcr-codebase/programming-elements/AreaOfCircle.java
package Dayone;
import java.util.*;
public class AreaOfCircle {
  public static void main(String [] args){
    //write a prgrm to calculate the area of a circle
    Scanner sc=new Scanner(System.in);
    //take radius value as input
    int radius=sc.nextInt();
    //take pi as input
    double pi=3.14;
    //calculate the area
    double area=pi*radius*radius;
    System.out.println("area of the circle is :" +area);

  }
}

File: core-java-practice/gcr-codebase/programming-elements/AreaOfCircle.cs
using System;
namespace ProgrammingElements{
    class AreaOfCircle{
        static void Main(string[] args){
            // write a program to calculate the area of a circle
            // take radius value as input
            int radius = int.Parse(Console.ReadLine());
            // take pi as value
            double pi = 3.14;
            // calculate the area
            double area = pi * radius * radius;
            Console.WriteLine("Area of the circle is: " + area);
        }
    }
}
